packages/destination-actions/src/destinations/klaviyo/orderCompleted/index.ts

```diff
@@ -21,7 +21,7 @@ const createEventData = (payload: Payload) => ({
         data: {
           type: 'metric',
           attributes: {
-            name: 'Order Completed'
+            name: payload.event_name
```

Comment:
Should we do something like this? Since event_name is not a required field and we are updating an existing destination?  
```suggestion
            name: payload.event_name ?? 'Order Completed'
```